test/e2e/conformance_test.go

```diff
@@ -105,6 +106,7 @@ var _ = Describe("conformance tests", func() {
 	}, 1)
 
 	AfterEach(func() {
+		setEnvVar("USE_CI_ARTIFACTS", "false", false)
```

Comment:
Similar here, what is the need for overriding the env variable here rather than using the value provided by Prow.